administrator/templates/atum/error_full.php

```diff
@@ -92,18 +166,18 @@
 <?php // Header ?>
 <header id="header" class="header">
 	<div class="d-flex">
-		<div class="header-title d-flex mr-auto">
+		<div class="header-title d-flex">
 			<div class="d-flex">
-				<?php // No home link in edit mode (so users can not jump out) and control panel (for a11y reasons) ?>
-				<div class="logo">
-					<img src="<?php echo $siteLogo; ?>" alt="<?php echo $logoAlt; ?>">
-					<img class="logo-small" src="<?php echo $smallLogo; ?>" alt="<?php echo $logoSmallAlt; ?>">
-				</div>
+				<a class="logo" href="<?php echo Route::_('index.php'); ?>"
+				   aria-label="<?php echo Text::_('TPL_BACK_TO_CONTROL_PANEL'); ?>">
+					<img src="<?php echo $siteLogo; ?>" alt="">
+					<img class="logo-small" src="<?php echo $smallLogo; ?>" alt="">
+				</a>
 			</div>
 			<jdoc:include type="modules" name="title"/>
 		</div>
-		<div class="header-items d-flex ml-auto">
-			<jdoc:include type="modules" name="status" style="header-element"/>
+		<div class="header-items d-flex">
+			<jdoc:include type="modules" name="status" style="header-item"/>
```

Comment:
Not sure if we have anything in the codestyle docs on this but everywhere else it seems to be 
`<jdoc:include ........" />`

Reply:
I created an issue: https://github.com/joomla/backend-template/issues/327